python-regression/tests/features/machine2/2_transaction_tests.feature

```diff
@@ -87,3 +87,106 @@ Feature: Test transaction confirmation
+        |value                  |11                         |int            |
+        |tag                    |VALUE9TRANSACTION          |string         |
+
+        #In the default test, the latest sent index will be 52. The next milestone issued should be 53.
+        When a milestone is issued with index 53 and references:
+        |keys                   |values                     |type           |
+        |transactions           |previousTransaction        |responseValue  |
+
+        #Give the node time to solidify the milestone
+        And we wait "15" second/seconds
+        
+        Given "getBalances" is called on "nodeA-m2" with:
+        |keys                   |values                     |type           |
+        |addresses              |FAKE_SPEND_ADDRESSES       |staticList     |
+
+        Then the response for "getBalances" should return with:
+        |keys                   |values                     |type           |
+        |balances               |0                          |int            | 
+
+    Scenario: Double spend only affects the ledger once
+        We want to ascertain that ledger state is always calculated correctly.
+        Even in the presence of double spend, the confirmed state should have spent only once
+
+        Then "1" transaction is issued on "nodeA-m2" with:
+        |keys                   |values                     |type           |
+        |address                |TEST_ADDRESS               |staticValue    |
+        |value                  |0                          |int            |
+        |tag                    |ZERO9VALUE                 |string         |
+
+        Then a double spend is generated referencing the previous transaction with:
+        |keys                   |values                     |type           |
+        |seed                   |DOUBLE_SPEND_SEED          |staticValue    |
+        |value                  |1000                       |int            |  
+        |tag                    |FAKE9VALUE                 |string         |
+
+        #In the default test, the latest sent index will be 53. The next milestone issued should be 54.
+        When a milestone is issued with index 54 and references:
+        |keys                   |values                     |type           |
+        |transactions           |firstDoubleSpend           |responseValue  |
+
+        #Give the node time to solidify the milestone
+        And we wait "15" second/seconds
+        
+        Given "getBalances" is called on "nodeA-m2" with:
+        |keys                   |values                     |type           |
+        |addresses              |DOUBLE_SPEND_ADDRESSES     |staticList     |
+
+        Then the response for "getBalances" should return with:
+        |keys                   |values                     |type           |
+        |balances               |1000 0                     |intList        | 
```

Comment:
what is this trailing 0?

Reply:
its an intList, we test the balance of 2 addresses, hence receiving 2 values